clientconn.go

```diff
@@ -1515,26 +1505,16 @@ func (ac *addrConn) getReadyTransport() transport.ClientTransport {
 // getTransport waits until the addrconn is ready and returns the transport.
 // If the context expires first, returns an appropriate status.  If the
 // addrConn is stopped first, returns an Unavailable status error.
```

Comment:
This comment needs updating now.

Reply:
I'll do one better and just delete the function.